### ISSUE TITLE
README is erroneous

### ISSUE THREAD

**Issue by Sarci1** [LabVIEW-DCAF/SystemLinkFileModule#1]
Seems to be copied from the SystemLink Tag Module.

**Comment by becega** [comment]
Yes this module was build from the code in the SystemLink Tag Module. I forgot to update the readme when I made this module
